index.html

```diff
@@ -6,7 +6,7 @@
         <meta charset="utf-8">
         <title>FOSSASIA | Google Code-In 2017</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">
+        <link href="css/https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">
```

Comment:
@duttaditya18 that's not how it's supposed to be, please check.

Reply:
Fixed.